examples/dht_server.rs

```diff
@@ -241,64 +239,3 @@ fn main() {
     info!("server running on localhost:12345");
     tokio::run(server);
 }
-
-fn run_server(server_obj: &Server) -> IoFuture<()> {
-    let interval = Duration::from_secs(1);
-    let dht_wakeups = Interval::new(Instant::now(), interval);
-    let mut server_obj_c = server_obj.clone();
-    let mut bootstrap_fast: bool = false;
-
-    let future = dht_wakeups
-        .map_err(|e| Error::new(ErrorKind::Other, format!("Nodes timer error: {:?}", e)))
-        .for_each(move |_instant| {
-            trace!("DHT server wake up");
-            // flag for fast bootstrapping
-            if bootstrap_fast {
-                server_obj_c.dht_main_loop()
-            } else {
-                bootstrap_fast = true;
```

Comment:
You missed why Namsoo played with `bootstrap_fast` and config params. Run dht server and you will see that the bootstrap happens only on the ~20 second.

Reply:
Yes, but it's not the best way to fix this delay, and I'm not sure that 20s delay worth it. I thought about making `last_nodes_req_time` variable optional with `None` initial value so we can see that we should send request immediately. But left this to another pr, since I think we should review and refactor bootstrap process - it has a lot to improve.

Reply:
> I'm not sure that 20s delay worth this juggling with config params

20s delay is too much.

Reply:
ok, made it optional. It seems it's not the best fix either, but butter than this magic with params.